common/lib/xmodule/xmodule/x_module.py

```diff
@@ -361,6 +362,14 @@ def xblock_kvs(self):
         self.save()
         return self._field_data._kvs  # pylint: disable=protected-access
 
+    @lazy
+    def _unwrapped_field_data(self):
+        """
+        This property hold the value _field_data here before we wrap it in
+        the LmsFieldData or OverrideFieldData classes.
+        """
+        return self._field_data
```

Comment:
@cpennington you mean like [for video descriptors](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/video_module/video_module.py#L322) and [while importing libraries](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/xml.py#L937). Is it enough to call `lazy.invalidate` once those are set?